pkgs/development/python2-modules/pygtk/default.nix

```diff
@@ -24,6 +24,8 @@ buildPythonPackage rec {
     })
   ];
 
+  outputs = [ "out" "dev" ];
```

Comment:
this probably breaks the use of this package in another python package. Can users of this package still find gtk2 pygobject2 pycairo ?